providers/openebs/cspc_csi_upgrade/README.md

```diff
@@ -0,0 +1,27 @@
+## Test Case to upgrade the cspc csi components
+
+## Description
+   - This TestCase is capable of Upgrade the OpenEBS CSPC CSI related components.
+   - First it will verify the CSPC CSI components version to upgrade.
+   - This test case upgrades the cstor operator, csi provisioner , cspc pool and csi volumes 
+     into new desired version. And verify the status of each components.
+   - This test constitutes the below files.
+     - cstor-csi-volume-upgrade-job.j2 - CSI volume upgrade job template which has to be populated with the given variables
+     - cstor-cspc-upgrade-job.j2 - CSPC pool upgrade job template which has to be populated with the given variables
+     - test_vars.yml - This test_vars file has the list of test specific variables used in LitmusBook
+     - test.yml - Playbook where the test logic is built to Upgrade the CSPC CSI components
+   - This test case the parameters in form of job environmental variables in run_litmus_test.yml
```

Comment:
This test case should be provided with the parameters in form of job environmental variables in run_litmus_test.yml